sources/platform/actors/development/actor_definition/input_schema/specification.md

```diff
@@ -181,18 +181,60 @@ Rendered input:
 
 ![Apify Actor input schema - country input](./images/input-schema-country.png)
 
+Example of date selection using absolute and relative `datepicker` editor:
+
+```json
+{
+    "absolute_date": {
+        "title": "Date",
+        "type": "string",
+        "description": "Select date",
+        "editor": "datepicker"
+    },
+    "relative_date": {
+        "title": "Relative date",
+        "type": "string",
+        "description": "Select date",
+        "editor": "datepicker",
+        "allowAbsolute": false,
+        "allowRelative": true
+    },
+    "any_date": {
+        "title": "Any date",
+        "type": "string",
+        "description": "Select date",
+        "editor": "datepicker",
+        "allowRelative": true
+    }
+}
+```
+
+Rendered input for `absolute_date` property. See that by default, the date picker allows selecting only an absolute date in the form `YYYY-MM-DD`:
```

Comment:
```suggestion
The `absolute_date` property renders a date picker that allows selection of a specific date in the format `YYYY-MM-DD`.
```

Reply:
Also, I'd stress here that this is the value returned by the field. See comment bellow.